### PR TITLE
Completed all the desired features.

### PR DESCRIPTION
**Pull request by jorgecuesta**
1. Full validation of config.json
2. Integration of WTS api using graphql generated
3. Added docker and docker-compose
4. Added github ci to generate the images
5. Updated README.md with more details about this

### CODE REVIEW

File: config.go [hunk 1]
```diff
@@ -2,127 +2,372 @@ package wtsc
...
 
 // LoadConfig loads and returns the configuration by reading the `configPath` file.
 func LoadConfig() *Config {
 	cfg := Config{}
-	var configPath = "./config.json"
+	var configPath = GetConfigFilePath()
+	Logger.Info().Str("path", configPath).Msg("reading config file")
 	var jsonFile *os.File
 	var bz []byte
 	if _, err := os.Stat(configPath); err != nil && os.IsNotExist(err) {
```
Comment (Alann27) [inline config.go:148]: Shouldn't it log a Fatal If the error is not a `os.IsNotExist` too?

File: config.go [hunk 2]
```diff
@@ -2,127 +2,372 @@ package wtsc
...
 	if err != nil {
-		log.Fatal().Str("configPath", configPath).Err(err).Msg("failed to close config file")
+		Logger.Fatal().Str("configPath", configPath).Err(err).Msg("failed to close config file")
 	}
 	err = json.Unmarshal(bz, &cfg)
 	if err != nil {
-		log.Fatal().Str("configPath", configPath).Err(err).Msg("failed to unmarshal config file")
-	}
-
-	err = ValidateConfig(&cfg)
```
Comment (Alann27) [inline config.go:45]: Why did you remove the ValidateConfig?

File: README.md [hunk 3]
```diff
@@ -1,19 +1,71 @@
...
+  "chain_pool": ["0021", "0003"], // Chain IDs available in your fleet
+  "servicer_keys": ["PRIVATE_KEY"], // List of private keys for signing stake transactions
+  "stake_weight": 4, // Used by the "What to Stake" service to estimate potential rewards (1-4)
+  "min_increase_percent": 5, // Minimum percentage increase expected to process stakes
+  "min_service_stake": [ // Minimum number of nodes for specific services
+    {
+      "service": "0021",
+      "min_node": 1
+    }
+  ],
```
Comment (Alann27) [inline README.md]: `chain_pool` should not be named `service_pool`? Because in the code, chain is always named service and I think is better to just have one name for the same thing.

**Review by Alann27 — CHANGES_REQUESTED** [review]

File: config.go (hunk 2 above)
Comment (jorgecuesta) [reply config.go:45]: was moved out on every place it is called, but now I move it at the very end of LoadConfig, previously was not trigger Fatal() on hot reload of config file to prevent process stop due to a bad change.

File: config.go (hunk 1 above)
Comment (jorgecuesta) [reply config.go:148]: Logger.Fatal is there.

File: README.md [hunk 4]
```diff
@@ -27,42 +27,39 @@ Before using the provided recommendations, it's important to understand several
...
+| poktscan_api          | string              | POKTscan API endpoint                                                                                            |
+| poktscan_api_token    | string              | POKTscan API token                                                                                               |
+| network_id            | string              | Can be "mainnet" or "testnet"                                                                                    |
+| tx_memo               | string              | Transaction memo                                                                                                 |
+| tx_fee                | integer             | Transaction fee                                                                                                  |
+| domain                | string              | Your node's domain (e.g., "poktscan.cloud" or "c0d3r.org")                                                       |
+| chain_pool            | array of strings    | Chain IDs available in your fleet                                                                                |
```
Comment (Alann27) [inline README.md]: chain_pool still here, replace it with service_pool

**Review by Alann27 — CHANGES_REQUESTED** [review]

**Review by Alann27 — APPROVED** [review]
